Azure/sonic-buildimage
# Support building multi-asic component

- What I did

1. Move single instance services into their own folder
2. Generate Systemd templates for any multi-instance service files in slave.mk
3. detect single or multi-instance platform in systemd-sonic-generator based on asic.conf platform specific file.
5. update container hostname after creation instead of during creation (docker_image_ctl)
6. run Docker containers in a network namespace if specified
7. add a service to create a simulated multi-ASIC topology on the virtual switch platform

- How to verify it
Single asic build was generated and loaded on the device to make sure the changes don't impact single asic image.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
